docs/contributing/governance.md

```diff
@@ -0,0 +1,59 @@
+# Governance
```

Comment:
```suggestion
```

You'll need to remove this and instead add the doc to the sidebar links list, then the title of that will be "Governance"